package.json

```diff
@@ -18,7 +18,7 @@
     "watch:firefox-local": "npm run build:firefox-local -- --watch",
     "watch:safari": "npm run build:safari -- --watch",
     "open:chrome": "./script/open-in-chrome ./dist/chrome https://github.com/bitcrowd/tickety-tick",
-    "open:firefox": "web-ext run --source-dir ./dist/firefox --pref startup.homepage_welcome_url=https://github.com/bitcrowd/tickety-tick",
```

Comment:
Sadly we need an addon-id to locally test out firefox now. So I switched the `open:firefox` task to `open:firefox-local` since the extensions breaks when doing `open:firefox`.

Reply:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID#When_do_you_need_an_add-on_ID